packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-action.ts

```diff
@@ -98,10 +97,10 @@ export function moveRadiusPoint(destination: vec.Vector2): MoveRadiusPoint {
 export const CHANGE_SNAP_STEP = "change-snap-step";
 export interface ChangeSnapStep {
     type: typeof CHANGE_SNAP_STEP;
-    snapStep: MafsGraphProps["snapStep"];
+    snapStep: [x: number, y: number];
```

Comment:
Question: Why were these changed to be explicit?

Reply:
My change to the prop type names broke the import of `MafsGraphProps`, and I took the opportunity to inline the types rather than fix the import.

My reasoning is: if you want to dispatch an action, you need to know what values to pass as the arguments. A type like `MafsGraphProps["snapStep"]` doesn't tell you what value you can pass — you have to take the extra step of looking up what `MafsGraphProps["snapStep"]` actually is.

The inlined types are more self-documenting. The downside, arguably, is that the types are now duplicated. But this duplication is the harmless kind. Normally, the problem with duplication is that things can get out of sync. But there's no danger that types will get out of sync because we have a tool (the typescript compiler) that ensures they stay consistent.